includes/models/class-listing.php

```diff
@@ -621,7 +621,7 @@ public function get_renewal_url( $deprecated = 0 ) {
 	 */
 	public function get_payment_url() {
 		$payment = $this->get_latest_payment();
-		return $payment->get_checkout_url();
+		return is_object( $payment ) && method_exists( $payment, 'get_checkout_url' ) ? $payment->get_checkout_url() : null;
```

Comment:
The return type for this function is a string. Can we return '' instead of null to keep the same return type?

Reply:
Done.

Reply:
Thank you!